x/zoneconcierge/types/params.go

```diff
@@ -4,6 +4,10 @@ import (
 	"fmt"
 )
 
+var (
+	EnableIntegration = false
```

Comment:
this is not a const, because it can be modified by tests?

Reply:
Yes this is intentional. I wanted to enable tests when the build tag `e2e` is enabled. This way we can keep all timestamping e2e tests running while disabling integration in real deployments